.travis.yml

```diff
@@ -1,5 +1,7 @@
-sudo: false
+sudo: true
 language: ruby
+before_install:
+  - sudo apt-get install -qq augeas-tools augeas-lenses libaugeas-dev
```

Comment:
http://docs.travis-ci.com/user/migrating-from-legacy/#Adding-APT-Packages mentions a way to install packages even without sudo. All of the above packages are in the Ubuntu precise whitelist so maybe worth a try?